core/src/test/scala/org/apache/spark/storage/BlockManagerSuite.scala

```diff
@@ -1987,6 +1992,10 @@ class BlockManagerSuite extends SparkFunSuite with Matchers with BeforeAndAfterE
     }
   }
 
+  test("SPARK-35589: test migration of index-only shuffle blocks during decommissioning") {
+    testShuffleBlockDecommissioning(None, true)
```

Comment:
As I see the test body here is the same as the one below ("test migration of shuffle blocks during decommissioning - no limit") so are we calling `testShuffleBlockDecommissioning(None, true)` twice with just a different test name. Is not it?

If I am right then we should come up with a common name for those two tests and keep only one with the new name as testing is already takes considerable time and we should avoid duplicating the same tests.

Reply:
Ya, you are correct. During refactoring to minimize the patch, I missed the duplication. I'll remove new test case.
> As I see the test body here is the same as the one below ("test migration of shuffle blocks during decommissioning - no limit") so are we calling testShuffleBlockDecommissioning(None, true) twice with just a different test name. Is not it?